core/backend/src/test/misc/GeoServices.test.ts

```diff
@@ -809,23 +809,31 @@ describe("GeoServices", () => {
     const validationRange = 50;
     const validationRangeSmall = 10;
 
-    const validateCRSList = async (expectedCount: number, allowedRange: number, extent?: Range2dProps): Promise<void> => {
-      const listOfCRS = await getAvailableCoordinateReferenceSystems({ extent });
+    interface CRSValidationOptions {
+      expectedCount: number;
+      allowedRange: number;
+      extent?: Range2dProps;
+      includeWorld?: boolean;
+    }
+
+    const validateCRSList = async (options: CRSValidationOptions): Promise<void> => {
+      const { expectedCount, allowedRange, extent, includeWorld } = options;
+      const listOfCRS = await getAvailableCoordinateReferenceSystems({ extent, includeWorld });
 
       // Check fields of returned CRS's
       const extentRange: Range2d = Range2d.fromJSON(extent);
       for (const crs of listOfCRS) {
-        // Validate extent
-        if (extent !== undefined) {
-          const crsExtentRange: Range2d = Range2d.fromJSON(crs.crsExtent);
-          const intersects = extentRange.intersectsRange(crsExtentRange);
-          assert.isTrue(intersects);
-        }
+      // Validate extent
+      if (extent !== undefined) {
+        const crsExtentRange: Range2d = Range2d.fromJSON(crs.crsExtent);
+        const intersects = extentRange.intersectsRange(crsExtentRange);
+        assert.isTrue(intersects);
+      }
 
-        // These fields should always be present
-        assert.isTrue(crs.name !== undefined);
-        assert.isTrue(crs.description !== undefined);
-        assert.isTrue(crs.deprecated === true || crs.deprecated === false);
+      // These fields should always be present
+      assert.isTrue(crs.name !== undefined);
+      assert.isTrue(crs.description !== undefined);
+      assert.isTrue(crs.deprecated === true || crs.deprecated === false);
```

Comment:
I believe that this unindent was unintentional.

Reply:
It wasn't, the formatting is better here. But for the sake of a cleaner history, I'll fix.

Reply:
Lines 826 through 836 can't be at the same indent level as the `for` statement on line 825.

Reply:
Should be fine now